src/vca_reviews/mod.rs

```diff
@@ -15,63 +13,27 @@ pub enum Error {
     CouldntParseTag(String),
 }
 
-#[derive(Clone, Deserialize)]
-struct AdvisorReviewRow {
-    id: u32,
-
-    triplet_id: String,
-
-    #[serde(alias = "Idea Title")]
-    title: String,
-
-    #[serde(alias = "Idea URL")]
-    url: String,
-
-    proposal_id: u32,
-
-    #[serde(alias = "Question")]
-    question: String,
-
-    question_id: u32,
-
-    #[serde(alias = "Rating Given")]
-    rating: u32,
-
-    #[serde(alias = "Assessor")]
-    assessor: String,
-
-    #[serde(alias = "Assessment Note")]
-    note: String,
-}
-
 impl AdvisorReviewRow {
-    fn as_advisor_review(&self, tags_map: &TagsMap) -> Result<AdvisorReview, Error> {
-        Ok(AdvisorReview {
-            id: self.id as i32,
```

Comment:
As I see this field `id` is not used, can we remove it ?
(I know that it is from the dependecy lib)

Reply:
Yeah, it is not there anymore. I think this was commented before the rebase maybe?

Reply:
Looks it is still present https://github.com/input-output-hk/catalyst-toolbox/blob/f88382435f93e97f7bc6642a75a0b3cb4c1e7c44/src/vca_reviews/mod.rs#L19 .

Reply:
Aha, you meant from the `AdvisorReview` itself. I do not think it is possible to remove, since this structure (iirc) is used for serialization on other means (where it does actually need the id).